readme.md

```diff
@@ -1,71 +1,66 @@
-## PyTMX
-##### For Python 3.7+
-
-If you have any problems or suggestions, please open an issue.
-
-If you find this useful, please consider making a donation to help support it
-https://liberapay.com/ltheden/donate
-
-Discord!  https://discord.gg/2taTP4aYR6
-
-*Released under the LGPL v3*
-
-### See the "apps" folder for example use and cut/paste code.
-
-News
+<div align="center">
```

Comment:
ok, so i like the way the centered readme looks, but i didn't realize that html tags are required to make that happen.  i don't want to make the readme look like a webpage when viewed in a text editor.  pure markdown is preferred here because it looks good when viewed by a human in a text editor.  adding all the html makes is look busy and i would prefer to leave it out, even if that means the nice formatting is lost.